metrics/shoveler.go

```diff
@@ -286,6 +277,12 @@ func LaunchShoveler(ctx context.Context, egrp *errgroup.Group, metricsPort int)
 			}
 			shoveler.PacketsReceived.Inc()
 
+			err = handlePacket(buf[:rlen])
+			if err != nil {
+				shovelerLogger.Errorln("Failed to handle packet:", err)
```

Comment:
Can you add a little more context to these `Failed to handle packet` log lines? I'm not 100% sure what they should say, but it'd be useful to log a) what pelican component should be responsible for handling the packet (shoveler? metrics engine? Not sure what's correct to say) and b) something about the packet we were expecting. Maybe the log should read something like:
```
shoveler.ErrorLn("Pelican failed to handle a monitoring UDP packet from the shoveler:", err)
```